src/functionalTest/resources/scenarios/RIA-3846-do-not-set-organisation-policy-start-appeal-flag-off.json

```diff
@@ -0,0 +1,28 @@
+{
+  "description": "RIA-3846 Do not set OrganisationPolicy at start appeal - feature flag off",
+  "request": {
```

Comment:
there is a need to add here `"launchDarklyKey": "share-case-feature:false"` when we enable share a case, test will fail by default

Reply:
Added.